client/app/scripts/superdesk-packaging/packaging.js

```diff
@@ -601,6 +609,43 @@
         };
     }
 
+    PackagingEmbeddedDirective.$inject = ['packages'];
+    function PackagingEmbeddedDirective(packages) {
+        return {
+            templateUrl: 'scripts/superdesk-packaging/views/packaging.html',
+            require: '^sdAuthoringContainer',
+            scope: {
+                item: '='
+            },
+            link: function(scope, elem, attrs, authoringCtrl) {
+                scope.$watch('item', function(item) {
+                    if (item && item.lockIt){
+                        scope.lock();
+                    } else {
+                        scope.origItem = null;
+                        scope.$applyAsync(function() {
```

Comment:
PackagingEmbeddedDirective is slightly changed copy of AuthoringEmbeddedDirective, and that part comes from there. You might be right, I'm not sure why it's done like this, possibly to deal with an unrelated problem, though I can't imagine what at the moment. @petrjasek might know better as he worked on the original directive.